src/pages/chainlink-functions/resources/service-limits.md

```diff
@@ -6,20 +6,20 @@ title: "Chainlink Functions Service Limits"
+| Request fulfillment timeout                                   | The maximum duration of an in-flight Chainlink Functions request. After this duration, the request is canceled | 5 minutes                                                                                                                       |
+| Maximum request (combined source code, args and secrets) size | The maximum size of a Chainlink Request. This includes the source code, arguments, and secrets                 | 30 kilobytes                                                                                                                    |
+| Maximum returned value size                                   | The maximum size of the value returned by the DON when fulfilling a request                                    | 256 bytes                                                                                                                       |
+| Maximum source code execution time                            | The maximum amount of time that a source code can run                                                          | 10 seconds                                                                                                                      |
+| Maximum memory allocated to the source code                   | The maximum amount of memory allocated to your source code during execution                                    | 128 megabytes                                                                                                                   |
+| HTTP - Maximum queries                                        | The maximum number of HTTP requests that your source code can make                                             | 5                                                                                                                               |
+| HTTP - query timeout                                          | The duration of an HTTP request before it times out                                                            | 3 seconds                                                                                                                       |
+| HTTP - Maximum URL length                                     | Length of the HTTP URL                                                                                         | 2048 characters                                                                                                                 |
+| HTTP - Maximum request length (body + headers)                | The maximum size of an HTTP request, including the request body and HTTP headers                               | 2 kilobytes                                                                                                                     |
```

Comment:
Similarly - we have "body+headers" in the description so maybe remove from the first column?